service/routes.py

```diff
@@ -64,6 +64,18 @@ def get_order(order_id):
     return make_response(jsonify(order.serialize()), status.HTTP_200_OK)
 
 
+######################################################################
+# List Orders
+######################################################################
+@app.route("/orders", methods=["GET"])
+def list_orders():
+    """List all of the Orders"""
+    app.logger.info("Request for order list")
+    orders = Order.all()
+    results = [order.serialize() for order in orders]
```

Comment:
Please add an info log here to indicate how many records are returned by this API response:

`app.logger.info("Returning %d orders", len(results))`

Reply:
done